cmd/ltagent/init.go

```diff
@@ -50,14 +50,7 @@ func genData(lt *loadtest.LoadTester, numUsers int64) error {
 		mlog.Info("loadtest done", mlog.String("elapsed", time.Since(start).String()))
 	}(time.Now())
 
-	for lt.Status().NumUsersAdded != numUsers {
-		if _, err := lt.AddUsers(10); err != nil {
-			return fmt.Errorf("failed to add users %w", err)
-		}
-		time.Sleep(5 * time.Second)
-	}
```

Comment:
Maybe I am missing something, but if you don't add users, how will you have multiple controllers?

Reply:
We are setting the `InitialActiveUsers` to the number of users given. The difference is that now they'd start immediately, while before we'd ramping them up in batches.

Reply:
My bad. `lt.Run` has that logic internally. Sorry, a bit late here. My brain isn't working clearly :)